pages/Signup.vue

```diff
@@ -1,58 +1,51 @@
 <template>
-  <!DOCTYPE html>
-  <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-  </head>
-  <body>
-    <div class="content-box">
-
-      <div class="catch-line-outer">
-          <div class="catch-line-inner">
-              <p>"Gear Up, Sign Up, Shop Now!"</p>
-          </div>
-      </div>
-      <div class="username">
-        <p>Username</p>
-        <input type="text" name=""  placeholder="Enter your name"/>
-      </div>
-      <div class="email">
-        <p>Email</p>
-        <input type="email" name="" placeholder="Enter your email"/>
-      </div>
-      <div class="password">
-        <p>Password</p>
-        <input type="password" name="" placeholder="*************"/>
-      </div>
-      <div class="sign-up">
+
+<body>
+  <div class="content-box">
+    <div class="catch-line-outer">
+        <div class="catch-line-inner">
```

Comment:
Is there a reason behind choosing `catch-line` for naming? What does it mean ?

Reply:
while i was going through figma design i noticed it has two boxes like these
![2023-06-12 (1)](https://github.com/bsoc-bitbyte/GetIt/assets/111559776/5e854b6c-9b58-4529-8f77-a095320400b3)
![2023-06-12](https://github.com/bsoc-bitbyte/GetIt/assets/111559776/f1892df4-f470-473a-8af5-595cf57a5bc7)

so i added those two dives.